.github/workflows/container-build.yml

```diff
@@ -0,0 +1,53 @@
+name: Docker Image Build CI
+
+on:
+  push:
+    branches:
+    - $default-branch
+    - deploy-*
+    tags:
+    - v*.*.*
```

Comment:
```  branches:
    - main
    - deploy-*
    tags:
      - v*.*.*
 ```

Reply:
@alexgarel

Reply:
For yaml, it's ok also if we don't indent.